src/cms/views/regions/region_list_view.py

```diff
@@ -44,7 +44,7 @@ def get(self, request, *args, **kwargs):
         """
         regions = Region.objects.all()
         # for consistent pagination querysets should be ordered
-        paginator = Paginator(regions.order_by("created_date"), PER_PAGE)
+        paginator = Paginator(regions.order_by("name"), PER_PAGE)
```

Comment:
Please apply the sorting directly in the [Meta class](https://docs.djangoproject.com/en/3.2/ref/models/options/#django.db.models.Options.ordering) of the region model, not in the region list view. This way, all region querysets will be sorted, no matter where they're queried.

Reply:
```suggestion
        paginator = Paginator(regions, PER_PAGE)
```
Now you can remove the ordering in the paginator, because it's already sorted by default.